preload.js

```diff
@@ -62,6 +62,8 @@ window.CONSTANTS = {
   MAX_LOGIN_TRIES: 3,
   MAX_PASSWORD_LENGTH: 32,
   MAX_USERNAME_LENGTH: 20,
+  DEFAULT_PUBLIC_CHAT_URL: 'https://chat.lokinet.org',
```

Comment:
Why can't we use `config.get('defaultPublicChatServer')` like before?

Reply:
Done.